R/utilities.R

```diff
@@ -111,14 +111,14 @@ get_ssh_session = function(host="gphost01.bcgsc.ca"){
 }
 
 
-
-#' Get regions (bed format) from genes.
+#' Get regions from genes.
 #'
 #' @param gene_symbol Gene symbol (e.g BCL2).
```

Comment:
Update to indicate that a vector of one or more gene symbols can be provided (same goes for next argument)